### PR TITLE
liftOver.loc was not accessible in local data

### PR DESCRIPTION
**Pull request by yhoogstrate**

### CODE REVIEW

File: tools/extract/liftOver_wrapper.xml [hunk 1]
```diff
@@ -20,10 +20,7 @@
       <validator type="dataset_metadata_in_file" filename="liftOver.loc" metadata_name="dbkey" metadata_column="0" message="Liftover mappings are currently not available for the specified build." />
     </param>
     <param name="to_dbkey" type="select" label="To">
-      <options from_file="liftOver.loc">
-        <column name="name" index="1"/>
-        <column name="value" index="2"/>
-        <column name="dbkey" index="0"/>
+      <options from_data_table="liftOver">
```
Comment (bgruening) [inline tools/extract/liftOver_wrapper.xml:23]: I think @martenson would argue to increase the version number for this tool :)

**Comment by bgruening** [comment]
@yhoogstrate thanks, looks good to me, besides the version number.

**Comment by jmchilton** [comment]
More info ... http://dev.list.galaxyproject.org/liftOver-functional-tests-failed-tp4666645p4666693.html. It is clearly broken but I think people will argue it needs to remain broken so workflows continue to work. Maybe just increasing the tool version is enough.

Galaxy can load multiple versions of the same tool id directly in the tool panel - maybe we should just ship Galaxy with two lift overs.

**Comment by martenson** [comment]
@jmchilton or migrate it?

**Comment by bgruening** [comment]
Better migrate than having two distinct version.

**Comment by yhoogstrate** [comment]
I think the setup of the test-case is a bit weird. It shouldn't pass the to_dbkey as a dbkey, because this assumes that hg18ToPanTro2.over.chain is installed, while this file is not in test-data and neither it's uncommented in liftOver.loc.sample.

It would be more logical to add a conditional that also allows using custom chain files, and use that one for the tests. That way you can pass through a reference to a file instead of a dbkey. If this sound reasonable I am willing to work it out.

**Comment by jmchilton** [comment]
I've been thinking about this - and unless there is something I am missing I think we should just bump the version and fix the data table. So this as is plus a version bump to tools/extract/liftOver_wrapper.xml.

**Comment by bgruening** [comment]
:+1: 
I would like to merge this. Any complains? ping @martenson

**Comment by martenson** [comment]
Thank you for the fix @yhoogstrate 

to follow up: We are currently considering a tool migration for the next (15.10?) release.
https://github.com/galaxyproject/galaxy/issues/636

**Comment by jennaj** [comment]
This is on usegalaxy.org now and there seems to be a problem, see: https://github.com/galaxyproject/galaxy/issues/1039

**Pull request merged**